packages/nextjs/components/scaffold-eth/Contract/WriteOnlyFunctionForm.tsx

```diff
@@ -87,7 +87,7 @@ export const WriteOnlyFunctionForm = ({
   });
 
   return (
-    <div className="flex flex-col gap-3">
+    <div className={`flex gap-3 py-4 ${inputs.length === 0 ? "flex-row justify-between" : "flex-col"}`}>
```

Comment:
Should probably also check `functionFragment.payable`, cause now a payable function without inputs looks like this:
![image](https://user-images.githubusercontent.com/15879327/222185112-9d15662f-c0b5-40ba-8293-935b429a8b75.png)

Reply:
Tysm fixed it [a4a9913](https://github.com/scaffold-eth/se-2/pull/223/commits/a4a9913c056407bb971457927a58643819d8fe57)